lib/views/pages/events/events.dart

```diff
@@ -175,20 +181,24 @@ class _EventsState extends State<Events> {
         element['title'] == 'test' ||
         element['title'] == 'Talawa Conference Test' ||
         element['title'] == 'mayhem' ||
-        element['title'] == 'mayhem1'); //dont know who keeps adding these
+        element['title'] == 'mayhem1' ||
+        element['organization']['_id'] !=
+            currentOrgID); //dont know who keeps adding these
     // This removes all invalid date formats other than Unix time
     eventList.removeWhere(
-        (element) => int.tryParse(element['startTime'].toString()) == null);
+        (element) => int.tryParse(element['startTime'] as String) == null);
     eventList.sort((a, b) {
       return DateTime.fromMicrosecondsSinceEpoch(
-              int.parse(a['startTime'].toString()))
+              int.parse(a['startTime'] as String))
           .compareTo(DateTime.fromMicrosecondsSinceEpoch(
-              int.parse(b['startTime'].toString())));
+              int.parse(b['startTime'] as String)));
     });
     eventsToDates(eventList, DateTime.now());
     setState(() {
       displayedEvents = eventList;
     });
+    // print('orgID ==== $currentOrgID');
```

Comment:
remove code added for debugging.

Reply:
I have removed the debug print code now @Sagar2366 sir.